bundles/org.openhab.binding.heliosventiremote/src/main/java/org/openhab/binding/heliosventiremote/internal/HeliosVentiRemoteVariable.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.heliosventiremote.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+
+/**
+ * The {@link HeliosVentiRemoteVariable} is a description of a variable in the Helios ventilation system.
+ *
+ * @author Raphael Mack - Initial contribution
+ */
+@NonNullByDefault
+public class HeliosVentiRemoteVariable {
+    public enum type {
+        Temperature,
```

Comment:
Please use all upper-case names for enum constants and seperate the enum from this class.